### ISSUE TITLE
Make expression files match submission format

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#1059]
We have some expression files

[https://www.pombase.org/data/releases/latest/misc/gene_expression_table.tsv](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.pombase.org%2Fdata%2Freleases%2Flatest%2Fmisc%2Fgene_expression_table.tsv&data=05%7C01%7Cvw253%40universityofcambridgecloud.onmicrosoft.com%7Cd00bd8e4621e4a032cbe08db03833f58%7C49a50445bdfa4b79ade3547b4f3986e9%7C1%7C0%7C638107635115152893%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5grJDDymMKxCL7Ujmj6xWbHjgJi8wcVTSiQIBuQOxFE%3D&reserved=0)

but the export format seems to include and internal ID (PBO) which should be exported as the correct relation
to match the format described here
https://www.pombase.org/documentation/quantitative-gene-expression-data-bulk-upload-format
and here
https://www.pombase.org/documentation/qualitative-gene-expression-data-bulk-upload-format

(note currently I think that only quantitative are exported)

@afg1

**Comment by kimrutherford** [comment]
The PBO terms in the file are just "RNA level" and "protein level" so we can probably just remove the "term_id" column.

**Comment by ValWood** [comment]
I wonder why we added it, since we have the "type" in column 3?

**Comment by ValWood** [comment]
Sorry I'm confusing with the input file. I agree, we don't need the ID column since the type is expicit in "term name". SInce this coumn woud no longer be an "mini ontology" term (ony required for chado), it might be better to also relabel the column term_name to "type" so that it matches the coumn with the same data in the  submission files?
https://www.pombase.org/documentation/qualitative-gene-expression-data-bulk-upload-format
although in the input file we call it "protein" not "protein level"

https://www.pombase.org/documentation/qualitative-gene-expression-data-bulk-upload-format

 Since it's expression data, that the data will be "x level" can be assumed. So maybe we ony need to export 
type= protein note "term_name=protein level"

Also we call it "gene expression data", that shoud be more generically "expression data"

It's probably easier to chat about this...

**Comment by ValWood** [comment]
The decision is make the output file(s) match the input file format which is described here:

https://www.pombase.org/documentation/qualitative-gene-expression-data-bulk-upload-format

https://www.pombase.org/documentation/quantitative-gene-expression-data-bulk-upload-format

**Comment by kimrutherford** [comment]
Note to self: change code that draws the violin plots and change configuration for columns in advanced search and table download

**Comment by kimrutherford** [comment]
> The decision is make the output file(s) match the input file format which is described here:

Also:
 - check that there isn't a bug that causes the copies_per_cell column to only contain "ND".

**Comment by kimrutherford** [comment]
> check that there isn't a bug that causes the copies_per_cell column to only contain "ND".

There's no bug, but it is a bit silly having that column in our current file.  The `copies_per_cell` column in the exported file contains the `Range` column from the imported files (I should have called the column "`range`").  The Carpy and Marguerat input files don't have any values for "Range" so it's always "ND".

There will be a few values for `range` in the new export file with all 12 columns.

**Comment by kimrutherford** [comment]
> I think we talked about changing "ND" to "n/a". Should I do that or just leave the column blank when there's no data?

Apart from decided what to do in this case, the new export file code is implemented ready for tonight's load.

**Comment by ValWood** [comment]
I think leave it blank...

**Comment by kimrutherford** [comment]
The new export code is done.  I'll double check in the morning to make sure the file is generated correctly.  Then I'll worked on the README and the check that the docs are OK.

Example here: http://desktop.kmr.nz/local/full_gene_expression_table.tsv

**Comment by kimrutherford** [comment]
>  I'll double check in the morning to make sure the file is generated correctly. 

It all seems OK.

To keep the violin plots and query builder working, I've kept the code that exports the older format gene expression file.  I've renamed the file to `htp_gene_expression_table.tsv` here:
https://curation.pombase.org/dumps/latest_build/misc/

I've named the new format file `full_gene_expression_table.tsv`.  I'm thinking that we'll only include this new format file in the release directory and keep the HTP file for internal use.

**Comment by PCarme** [comment]
> The decision is make the output file(s) match the input file format which is described here:
> 
> https://www.pombase.org/documentation/qualitative-gene-expression-data-bulk-upload-format
> 
> https://www.pombase.org/documentation/quantitative-gene-expression-data-bulk-upload-format

Kind of related, I have modified the documentation pages as we had discussed. What do you think of this version ?

**Comment by ValWood** [comment]
Looks good. Maybe we should add a link to 
CHEBI https://www.ebi.ac.uk/chebi/
and 
QuickGO phase branch https://www.ebi.ac.uk/QuickGO/term/GO:0044848
to make it easier for people to  provide these?

**Comment by PCarme** [comment]
Makes sense. Although the GO terms used can also be cellular response terms or even cellular component terms, so it might be a bit restrictive to link only to the phase term.

**Comment by ValWood** [comment]
Good point.  Maybe just supply the root node for process.
